src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -44,21 +44,42 @@ class ProjectFileParser {
         FilenameUtils.removeExtension(file) + newExtension
     }
 
+    File getOutputPath() {
+        if (msbuild.destinationDir) {
+            project.file(msbuild.destinationDir)
+        }
+        if (properties.TargetDir) {
+            new File (properties.TargetDir)
+        }
+    }
+
     File getDotnetAssemblyFile() {
-        project.file(properties.TargetPath)

```

Comment:
could be because of parseProjectFile result is having solution as part of the result
as I am based on @adescamps-ullink 's pull request because my project parsing fails without that fix

Json reply is like below

{
    "Solution": {
    ...
    }
    "csproj": {
    ...
    }
}